README.md

```diff
@@ -1,2 +1,478 @@
+  - This could be `New York 3`, `San Francisco 1`, etc.
+
+If any of the information entered above information is inaccurate
+then warning messages should
+show up on the page for that site shortly after the first proxy is
+requested from that source.
+
+## Settings
+Environment variables stored in redis.  These are populated on deploy from
+config/default_settings.yml.
+- success_ratio_threshold
+  - Each proxy is periodically evaluated for its ability to be used on each
+  site.  If the success_count/(success_count+failure_count) is less than this
+  number, then the proxy may be decommissioned.
+- default_retry_interval
+  - How long to tell clients to wait for proxies to be created when we have
+  none.
+- server_retry_timeout
+  - How long to wait for newly provisioned proxies to be ready before giving up.
+  - These proxies aren't lost forever.  They can be reclaimed by the system
+  later.
+- proxy_age_timeout_seconds
+  - If proxies haven't been requested for a site in this many seconds then
+  the site forgets that it had ever used the proxies that are presently
+  allocated to it.
+  - If there are no more sites presently reserving these proxies then the
+  proxies are decommissioned.
+- failure_threshold
+  - If this many failures happen in between runs of the periodic performance
+  analyzer then evaluate this proxy's performance early.
+  - If the number of successes is too low then the proxy will be removed from
+  the site's proxy pool, and possibly decommissioned.
+
+## API Keys
+API keys can be created or destroyed.  Use the key in the api_key parameter
+with the API.
+
+# Installation and deploy instructions
+
+Before you deploy to production you'll need to create a development environment.
+Your development environment will only require some of the same steps as
+deploying to production.  Install the core apt-get dependencies and rvm.
+From there, run:
+```
+rvm install ruby2.2.0
+rvm use ruby2.2.0@zartan --create
+git clone https://github.com/Raybeam/zartan.git
+cd zartan
+gem install bundler
+bundle install
+```
+Create and modify the
+[config files](#config-files) for development.  Run `bundle exec rails s` to
+start the rails server, and `QUEUE=* rake environment resque:work` to start
+a resque worker.
+
+Zartan is a rails application which relies on a database, redis, resque and
+resque-scheduler.  The initial deployment of zartan runs on an ubuntu server,
+uses a postgres database, nginx/unicorn to run the rack workers and uses monit
+for process monitoring.  The database
+can be easily changed in config/database.yml.  The deploy
+procedure assumes that monit is being used for process monitoring, but monit
+is not required to actually run zartan.  The deploy procedure could be
+changed to use some other process monitoring service if necessary.
+
+The rest of the installation instructions will assume the above configuration
+unless otherwise specified.
+
+1. Initial setup. On the target machine, as "superuser":
```

Comment:
But we already know that this list of dependencies works for Ubuntu.  If someone were to install Zartan on an Ubuntu server, than the actual apt-get commands they need to run will be more useful than a list of package names.  I already have a disclaimer that these installation instructions assume Ubuntu.  Where do we draw the line for making assumptions about someone else's environment?  Do we add mysql, oracle, etc instructions?  Do we put Windows installation instructions?